iUAEParents/MainEmulationViewController.mm

```diff
@@ -326,4 +384,250 @@ - (void)dealloc
     [super dealloc];
 }
 
+/******* START MCSESSION mithrendal ****/
+extern int mainMenu_servermode;
```

Comment:
Sounds good thanks for doing this.

For the constant names, since they are used in objective-c, you can use this objective-c style: kConstName1, kConstName2 etc

> On Feb 25, 2016, at 21:35, mithrendal <notifications@github.com> wrote:
> 
> In iUAEParents/MainEmulationViewController.mm:
> 
> > @@ -326,4 +384,250 @@ - (void)dealloc
> >      [super dealloc];
> >  }
> >  
> > +/******* START MCSESSION mithrendal ****/
> > +extern int mainMenu_servermode;
> @simontoens 
> I am done with the separation. ;-) Moved nearly all MPC related stuff into a separate class file called MultiPeerConnectivityController.mm . Next is the constants thing for better readability of source code. I think I will create a .h file which declares an enum like this
> typedef enum MPCStateTypes
> {
> CONNECTION_IS_OFF = 0,
> SERVE_AS_HOST_FOR_INCOMING_JOYPAD_SIGNALS = 1,
> SEND_JOYPAD_TO_SERVERS_JOYSTICKPORT0 = 2,
> SEND_JOYPAD_TO_SERVERS_JOYSTICKPORT1 = 3
> } MPCStateType;
> 
> what do you think?
> 
> I will do the commit today or tomorrow at least on saturday ;-).
> 
> ―
> Reply to this email directly or view it on GitHub.
>